### ISSUE TITLE
Transferring a file from the desktop to the Files

### ISSUE THREAD

**Issue by Vadimmercury** [files-community/Files#4130]
**Describe the bug**
I can't transfer files from my desktop to Files. But from the program to the desktop it is possible to move the file

**To reproduce**
1. I open the program
2. I hook the file on the desktop with the mouse
3. I move to the "Files"
4. But the mouse cursor shows that moving is not possible

**Expected behavior**
I expect the file to move to a folder. But this does not happen

**Log file**
I do not attach the file log, since it does not show any errors

**Screenshots**
![Screenshot_5](https://user-images.githubusercontent.com/42690987/111477901-ff90b200-8748-11eb-9e62-ffa361ed40f7.png)

**System Information**
 - OS Version:  Windows 10 20H2 19042.867
 - App version:  v1.3 
 
**Additional context**
I am sorry if this is not a error. I would like to know if there is a solution to this problem

**Comment by gave92** [comment]
Thanks for this report. Could you check if it works when you set the folder layout mode to Grid or Tiles?

**Comment by Vadimmercury** [comment]
> Thanks for this report. Could you check if it works when you set the folder layout mode to Grid or Tiles?

No, unfortunately, when I change the folder layout mode the problem remains

**Comment by nt4f04uNd** [comment]
i wasn't able to reproduce, it works totally fine for me

app version 1.5.0.0
win 10 pro 2004

**Comment by Bainky** [comment]
> 
> 
> > Thanks for this report. Could you check if it works when you set the folder layout mode to Grid or Tiles?
> 
> No, unfortunately, when I change the folder layout mode the problem remains

I had the same problem and solved it by enabling UAC (Microsoft User Account Control).
If you disabled it, you can try to enable it;

- Press the key combination **WIN+R** and type `UserAccountControlSettings.exe`
- Drag the slider upwards like in the screenshot and then reboot your system.

![dllhost_N0RkLDOeLg](https://user-images.githubusercontent.com/61879874/112299991-91a53700-8cc2-11eb-8dae-c0cf942e4b82.png)

**Comment by Vadimmercury** [comment]
> > > Thanks for this report. Could you check if it works when you set the folder layout mode to Grid or Tiles?
> > 
> > 
> > No, unfortunately, when I change the folder layout mode the problem remains
> 
> I had the same problem and solved it by enabling UAC (Microsoft User Account Control).
> If you disabled it, you can try to enable it;
> 
> * Press the key combination **WIN+R** and type `UserAccountControlSettings.exe`
> * Drag the slider upwards like in the screenshot and then reboot your system.
> 
> ![dllhost_N0RkLDOeLg](https://user-images.githubusercontent.com/61879874/112299991-91a53700-8cc2-11eb-8dae-c0cf942e4b82.png)

Thank you so much!
Большущий респект, работает))
